roles/ceph-mon/tasks/docker/pre_requisite.yml

```diff
@@ -47,6 +53,13 @@
   tags:
     with_pkg
 
+- name: install six
```

Comment:
Unfortunately, I need to ensure that a certain version of six is installed, so I can't use the with_items loop.  I did add a comment to indicate this.